src/components/MoonLevels.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import { Moon } from './Moon'
+
+export const MoonLevels = () => {
+	const moons = Array.from(Array(13), (moon, level) => <Moon id={level} key={level} />)
```

Comment:
lol. Is this just another way around a for loop?

Reply:
I should actually update it. I really didn't want to use > ES6 code at the the time so this was a solution I found. 

I've since then found cleaner ways of doing this in my algo practices. 

```jsx
Array.from(18).fill(<component/>)
```

Moon also doesn't have to be there, I could have done (_,

Reply:
^yeah, I also noticed that moon didn't need to be there. Kinda why I went investigating